text.py

```diff
@@ -418,3 +418,93 @@ def result(self, state, action):
     def goal_test(self, state):
         """We're done when all letters in search domain are assigned."""
         return len(state) >= len(self.decoder.chardomain)
+#-----------------------------TF-IDF--------------------------------------
+class tf_idf_analysis:
+    '''a class to perform tf-idf analysis on a given set of documents and also search a query'''
+#     class variabels(their type)= values contained in the variable
```

Comment:
Add this part in the overall function comment, with an indentation between each line. Like this:

<pre>
"""
comment here

    * var_a: description
    * var_b: other description
"""
</pre>